conf/messages

```diff
@@ -419,8 +419,8 @@ propertyRepresentation.removeAgent.title=Remove agent permissions
 propertyRepresentation.removeAgent.message.p1=You can remove an agent''s ability to carry out checks and challenges on your behalf.
 propertyRepresentation.removeAgent.message.p2=Select an agent from the list below to remove them from one or more properties.
 propertyRepresentation.removeAgentProperties.title=Which properties should the agent be removed from
-propertyRepresentation.removeAgentProperties.heading=Remove an agent
-propertyRepresentation.removeAgentProperties.message.p1=You can remove {0}''s (Agent ID: {1}) ability to carry out checks and challenges for any of these properties:
+propertyRepresentation.removeAgentProperties.heading=Remove agent {0} from one or more properties
+propertyRepresentation.removeAgentProperties.message=(Agent ID: {0}) will no longer be able to submit checks and challenges for these properties.
```

Comment:
`(Agent ID: {0})`  is this really the text that's supposed to show ? I find it strange that it's in round brackets and that we show some ID to the user, rather than the agent name... I'm just checking, I find the brackets especially strange

Reply:
In the HTML, it has the agent name in bold, and uses this message afterwards, so the page does show the agent name and then the ID